### ISSUE TITLE
Scenario Annotation

### ISSUE THREAD

**Issue by albertosipov** [authorjapps/zerocode#258]
Hello Guys, Is it possible to add a `@Scenario` annotation to the framework?

> _`@Scenario` will pick the multi-step json and execute it... Usual stuff._

Basically I am looking for,
```
    @Test
    @Scenario("tests/customer_api_crud_tests.json")
    public void testCrudScenario() throws Exception {
    }
```

Let me know in case you need more info!

Thanks in advance!

**Comment by BartRobeyns** [comment]
Isn't that already covered by @JsonTestCase? It executes the scenario in the json-file.

**Comment by albertosipov** [comment]
#### _Note(Update) :_
Hello @BartRobeyns, `@JsonTestCase` does the identical job 👍 as highlighted.

We are migrating our Java Step definition based BDDs(Given-When-Then and glue codes) to Zerocode to minimize(ease) the test maintenance of the overall program.

A `@Scenario` is something very well understood by the team.

It's not blocking us, but a nice to have feature so that we do it from the beginning of this migration(retrofit).

Thanks for understanding!

**Comment by authorjapps** [comment]
Feature available in version 1.3.9 and higher.
